### PR TITLE
DB classes for registration form

### PR DESCRIPTION
**Pull request by AnnKirienko**

### CODE REVIEW

File: DAL/Repositories/UnitOfWork.cs [hunk 1]
```diff
@@ -94,7 +98,55 @@ public BaseRepository<Topic> Topics
                 return topicRep;
             }
         }
-        
+        public BaseRepository<Specialization> SpecializationRep
```
Comment (DShcharbakou) [inline DAL/Repositories/UnitOfWork.cs:101]: There are no postfix **Rep** in other repositories. I suppose need delete this postfix.

File: DAL/Repositories/UnitOfWork.cs [hunk 2]
```diff
@@ -94,7 +98,55 @@ public BaseRepository<Topic> Topics
...
+            {
+
+                if (this.specializationRep == null)
+                {
+                    this.specializationRep = new SpecializationRepository(internshipDbContext);
+                }
+                return specializationRep;
+            }
+        }
+        public BaseRepository<EnglishLevel> EnglishLevelRep
```
Comment (DShcharbakou) [inline DAL/Repositories/UnitOfWork.cs:113]: There are no postfix **Rep** in other repositories. I suppose need delete this postfix.

File: DAL/Repositories/UnitOfWork.cs [hunk 3]
```diff
@@ -94,7 +98,55 @@ public BaseRepository<Topic> Topics
...
+            {
+
+                if (this.englishLevelRep == null)
+                {
+                    this.englishLevelRep = new EnglishLevelRepository(internshipDbContext);
+                }
+                return englishLevelRep;
+            }
+        }
+        public BaseRepository<Country> CountryRep
```
Comment (DShcharbakou) [inline DAL/Repositories/UnitOfWork.cs:125]: There are no postfix **Rep** in other repositories. I suppose need delete this postfix.

File: DAL/Repositories/UnitOfWork.cs [hunk 4]
```diff
@@ -94,7 +98,55 @@ public BaseRepository<Topic> Topics
...
+            {
+
+                if (this.countryRep == null)
+                {
+                    this.countryRep = new CountryRepository(internshipDbContext);
+                }
+                return countryRep;
+            }
+        }
+        public BaseRepository<City> CityRep
```
Comment (DShcharbakou) [inline DAL/Repositories/UnitOfWork.cs:137]: There are no postfix **Rep** in other repositories. I suppose need delete this postfix.

**Review by DShcharbakou — APPROVED** [review]

**Review by wollentimer — APPROVED** [review]

**Review by VasiliNesterenko — APPROVED** [review]

**Pull request merged**